documentation/book/assembly-kafka-connect.adoc

```diff
@@ -16,15 +15,14 @@ The default image currently contains only the `FileStreamSinkConnector` and `Fil
 The REST interface for managing the Kafka Connect cluster is exposed internally within the {ProductPlatformName} cluster as a `kafka-connect` service on port `8083`.
 
 Example `KafkaConnect` resources and the details about the `KafkaConnect` format for deploying Kafka Connect can be found in
-xref:kafka_connect_config_map_details[].
+xref:assembly-deployment-configuration-kafka-connect-str[here]
```

Comment:
We should be more specific than 'here' - it should say 'Deployment of Kafka Connect' or whatever the title is.